src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] <b>REQUIRED:</b> The project requires `App` component to be class-based component. Kindly do that to comply with project requirements.